marshmallow/utils.py

```diff
@@ -334,16 +334,31 @@ def callable_or_raise(obj):
     return obj
 
 
-def get_func_args(func):
```

Comment:
Sorry, I still had the first commit in mind, where the functions were imported from utils.py into compat.py. Taking [your comment here](https://github.com/marshmallow-code/marshmallow/pull/548/commits/c971b98bf074481200c54d198f7e7e18a5ea359b#r87485599) into account, it makes sense.